indico/modules/events/registration/templates/management/email.html

```diff
@@ -1,4 +1,17 @@
 {% from 'forms/_form.html' import form_header, form_rows, form_footer  %}
+{% from 'message_box.html' import message_box %}
+
+{% if registrations_without_ticket_count %}
+    <div class="attach-ticket-warning" style="display: none;">
```

Comment:
I think we need to move this farther down - unless you are on at least a 1440p screen, you'll most likely have to scroll to see the switch, but then the warning will not be visible since it'll be in the area from which you scrolled away...